lib/wibox/layout/stack.lua

```diff
@@ -0,0 +1,166 @@
+---------------------------------------------------------------------------
+-- @author Emmanuel Lepage Vallee
+-- @copyright 2016 Emmanuel Lepage Vallee
+-- @release @AWESOME_VERSION@
+-- @classmod wibox.layout.stack
+---------------------------------------------------------------------------
+
+local base  = require("wibox.widget.base" )
+local fixed = require("wibox.layout.fixed")
+local table = table
+local pairs = pairs
+local floor = math.floor
+local util  = require("awful.util")
+
+local stack = {mt={}}
+
+--- Layout a fixed layout. Each widget gets just the space it asks for.
+-- @param layout The layout you are modifying.
+-- @param context The context in which we are drawn.
+-- @tparam number width The available width.
+-- @tparam number height The available height.
+-- @name layout
+-- @class function
+
+--- Get all direct children widgets
+-- @param layout The layout you are modifying.
+-- @return a list of all widgets
+-- @name get_children
+-- @class function
+
+--- Add some widgets to the given fixed layout
```

Comment:
This (and other docs) talk about `the given fixed layout`. Hm, and I bet the same occurs in `flex`....